src/efi.rs

```diff
@@ -139,7 +141,8 @@ impl Efi {
             log::debug!("Not booted via EFI, skipping firmware update");
             return Ok(());
         }
-        let product_name = get_product_name()?;
+        let sysroot = Dir::open_ambient_dir(&Path::new("/"), cap_std::ambient_authority())?;
```

Comment:
Minor nit, you can just pass `"/"` here, because the API accepts `impl AsRef<Path>`.